tests/functional/cli/container/test_container.py

```diff
@@ -84,6 +84,23 @@ def test_bucket_list(self):
         output = self.openio('bucket list ' + opts)
         self.assertIn(cname, output)
 
+    def test_bucket_list_with_versioning(self):
+        opts = self.get_format_opts(fields=('Name', ))
+        cname = 'mybucket-' + random_str(4).lower()
+        output = self.openio('container create --bucket-name %s %s %s' %
+                             (cname, cname, opts))
+        self.assertOutput(cname + '\n', output)
+
+        opts = self.get_format_opts(fields=('Name', 'Versioning'))
+        opts += " --prefix %s --versioning" % cname
+
+        output = self.openio('bucket list ' + opts)
+        self.assertIn("Suspended", output)
+
+        self.openio('container set --max-versions -1 %s' % cname)
```

Comment:
To be coherent, we can use `--versioning` instead of `--max-versions`.

Reply:
Done